merlin-server/constraints-dsl-compiler/src/libs/mission-model-generated-code.ts

```diff
@@ -25,3 +25,10 @@ export const ActivityTypeParameterMap = {
     })
   }),
 }
+
+export type ParameterTypeactivity = {
+  parameter: string
+}
+export type ActivityTypeParameterMap = {
+  [ActivityType.activity]: ParameterTypeactivity
+};
```

Comment:
two things:
- I don't think this is used in constraints, only scheduling. So shouldn't these types go in scheduling instead? I don't like how we have such a big distinction between the two, but as long as we have the distinction we might as well abide by it.
- If the `ParameterType<activity>` type isn't going to be used directly by itself, I'd prefer to inline it into `ActivityTypeParameterMap` to avoid polluting the namespace.

Reply:
Or if you don't want to inline the types, you can just not export them.

Reply:
1) a) Okay, I'll move it. b) I tried inlining but I could not make it work, that's why it is this way. Will try again now that pieces are not moving so much around. 
2) Okay

Reply:
I tried and this is going to take time, I'd rather not do it now.